src/admin-views/settings/emails/email-template/body/event-location.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+
+if ( empty( $event_venue ) || ! is_array( $event_venue ) ) {
+	return;
+}
+?>
+<tr>
+	<td style="padding:54px 0 12px 0">
+		<h3 style="font-size:16px;font-weight:700;background:transparent;padding:0;margin:0;">
+			Event Location
+		</h3>
+	</td>
+</tr>
+<tr>
+	<td style="border:1px solid #d5d5d5;padding:25px;">
+		<h2 style="font-size: 18px;font-weight: 700;margin:0;padding:0;background:transparent;">
+			<?php echo $event_venue['name']; ?>
+		</h2>
+		<table role="presentation" style="width:100%;border-collapse:collapse;border:0;border-spacing:0;">
+			<tr>
+				<td style="padding:12px 0 0 0;">
+					<table role="presentation" style="width:100%;border-collapse:collapse;border:0;border-spacing:0;">
+						<tr>
+							<td style="padding:0 25px 0 0;text-align:center;vertical-align:top;width:20px" valign="top" align="center">
+								<img width="20" height="28" style="width:20px;height:28px;display:block;" src="<?php echo plugins_url('/event-tickets/src/resources/icons/map-pin.svg') ?>" />
+							</td>
+							<td style="padding:0;text-align:left">
+								<?php echo $event_venue['address1']; ?><br>
+								<?php echo $event_venue['address2']; ?><br>
+								<a href="#">Get Directions</a>
+							</td>
+						</tr>
+					</table>
+				</td>
+				<td style="padding:0;">
+					<table role="presentation" style="width:100%;border-collapse:collapse;border:0;border-spacing:0;margin-bottom:18px">
+						<tr>
+							<td style="padding:0 20px 0 0;text-align:center;vertical-align:top;width:25px" valign="top" align="center">
+								<img width="25" height="24" style="width:25px;height:24px;display:block;" src="<?php echo plugins_url('/event-tickets/src/resources/icons/phone.svg') ?>" />
+							</td>
+							<td style="padding:0;">
+								<?php echo $event_venue['phone']; ?>
+							</td>
+						</tr>
+					</table>
+					<table role="presentation" style="width:100%;border-collapse:collapse;border:0;border-spacing:0;">
+						<tr>
+							<td style="padding:0 21px 0 0;text-align:center;vertical-align:top;width:24px" valign="top" align="center">
+								<img width="24" height="23" style="width:24px;height:23px;display:block;" src="<?php echo plugins_url('/event-tickets/src/resources/icons/link.svg') ?>" />
+							</td>
+							<td style="padding:0;">
+								<a href="<?php echo $event_venue['website']; ?>" target="_blank">
```

Comment:
We should probably move these atoms into its own template and show only if the information is available.

Reply:
Also, we should use the proper `rel="noopener noreferrer"` to any link which target is blank, for security purposes.